content/recipes/prisma.md

```diff
@@ -516,6 +515,17 @@ This controller implements the following routes:
 
 - `/post/:id`: Delete a post by its `id`
 
+#### Issues with `enableShutdownHooks`
+
+Prisma interferes with NestJS `enableShutdownHooks` - Prisma listens for shutdown signals and will call process.exit() before your application shutdown hooks fire. To deal with this, you would need to add a listener for Prisma `beforeExit` event. 
```

Comment:
Something here like `[Prisma listens for shutdown signals and will call process.exit()](issue-link-here)`

Reply:
So there's no 'direct' issue since its not a bug in the lib, but a feature. I've linked to the comment in a ticket that brings this up, which explains where the beforeExit hook comes from, so people can work from that ticket if they need more context? lmk